src/lib/db.js

```diff
@@ -6,12 +6,14 @@ const path = require('path')
 const connection = require('./knex').config.connection
 const spawn = require('child_process').spawn
 const knex = require('./knex').knex
+const sequence = require('./utils').sequence
 const readRejectionReasons = require('../models/db/rejectionReasons')
   .readRejectionReasons
 const readTransferStatuses = require('../models/db/transferStatuses')
   .readTransferStatuses
 
 const TABLE_NAMES = [
+  'L_TRANSFER_ADJUSTMENTS',
```

Comment:
No, but I think it's ok the leave it this way since it's simpler to just have a list of all table names.